test/spec-validation.js

```diff
@@ -0,0 +1,40 @@
+const testlab = require('@loopback/testlab');
+const createApp = require('../dist/application').createApp;
+const expect = testlab.expect;
+const validateApiSpec = testlab.validateApiSpec;
+
+describe('Binding', () => {
```

Comment:
Since this test file is testing OpenAPI spec, the `describe` block should be called "Application API spec" or similarly.